src/renderer/i18n/locales/it-IT.js

```diff
@@ -0,0 +1,964 @@
+    },
+
+    PROFILE_ALL: {
+      HEADER: 'I miei profili',
+      ADD_PROFILE: 'Aggiungi profilo',
+      EDIT_PROFILE: 'Modifica profilo',
+      REMOVE_PROFILE: 'Rimuovi questo profilo',
+      SELECT_PROFILE: 'Usa questo profilo'
+    },
+
+    PROFILE_NEW: {
+      STEP1: {
+        INSTRUCTIONS: {
+          HEADER: 'Crea un profilo',
+          TEXT: 'Inserisci il tuo nome o nickname e seleziona la lingua preferita e la valuta predefinita.'
+        },
+        NAME: 'Nome profilo',
+        TITLE: '1. Dettagli profilo',
+        AVATAR: 'Seleziona il tuo avatar preferito o la prima lettera del nome del tuo profilo',
+        NO_AVATAR: 'Nessun Avatar'
+      },
+      STEP2: {
+        INSTRUCTIONS: {
+          HEADER: 'Selezione rete',
+          TEXT: 'Seleziona la rete di questo profilo'
+        },
+        CUSTOM_NETWORK: 'oppure scegli una rete personalizzata',
+        CUSTOM_NETWORK_EXPLAIN: 'Puoi selezionare la tua rete personalizzata o scegliere una delle reti disponibili.',
+        NAME: 'Nome profilo',
+        TITLE: '2. Rete'
+      },
+      STEP3: {
+        INSTRUCTIONS: {
+          HEADER: 'Aspetto',
+          TEXT: 'Personalizza questa applicazione scegliendo uno dei nostri temi e sfondi'
+        },
+        MARKET_CHART: 'Scegliere di visualizzare il grafico dei prezzi sulla dashboard',
+        THEME: 'Puoi scegliere tra modalità chiara e modalità scura',
+        BACKGROUND: 'Scegli il tuo sfondo preferito',
+        NAME: 'Nome profilo',
+        TITLE: '3. Aspetto'
+      }
+    },
+
+    PROFILE_EDITION: {
+      TAB_PROFILE: {
+        INSTRUCTIONS: {
+          HEADER: 'Modifica questo profilo',
+          TEXT: 'Qui puoi modificare il tuo nome o il nickname, selezionare la lingua preferita e la valuta predefinita, la rete e l\'avatar.'
+        },
+        TITLE: 'Profilo'
+      },
+      TAB_DESIGN: {
+        INSTRUCTIONS: {
+          HEADER: 'Personalizza questo portafoglio',
+          TEXT: 'Qui puoi scegliere il tema di default (chiaro o scuro) e lo sfondo.'
+        },
+        TITLE: 'Design portafoglio'
+      }
+    },
+
+    WALLET: {
+      TRANSACTIONS: 'Transazioni',
+      DELEGATES: 'Delegati',
+      STATISTICS: 'Statistiche',
+      SIGN_VERIFY: 'Firma',
+      PURCHASE: 'Acquista {ticker}'
+    },
+
+    WALLET_EXCHANGE: {
+      CHANGELLY_TERMS: {
+        TITLE: 'Acquista {ticker} direttamente dall\'ARK Desktop Wallet tramite la nostra integrazione di Changelly',
+        CONTENT: 'Changelly ti permette di scambiare altre criptovalute in {ticker} ed ha un\'opzione per acquistare {ticker} direttamente tramite le tue carte di credito o di debito.',
+        CONFIRMATION: 'Ho letto e sono d\'accordo con i termini {terms} e {privacy} di Changelly. Facendo clic su {button}, riconosco e comprendo che la mia transazione potrebbe attivare la verifica AML/KYC in conformità con Changelly {kyc}.',
+        TERMS_OF_USE: 'termini d\'uso',
+        PRIVACY_POLICY: 'privacy policy',

```

Comment:
Is this correct?

Reply:
Almost always seen written in English also in Italian contexts. The literal and correct translation is:

PRIVACY_POLICY: 'Politica sulla riservatezza'